components/colors/colors-and-shadows.hbs

```diff
@@ -19,6 +19,13 @@
   {{/each}}
 </div>
 
+<h1>Transparent Colors</h1>
+<div class="color-container">  
+  {{#each transparent}}
+    <div class="color-square {{textColor}} {{backgroundColor}}">{{text}}<span class="variable-name">{{scssVarName}}</span></div>
```

Comment:
What is scssVarName ?

Reply:
scssVarName is a property in the .yml file, used to specify the scss variable name coming from our styles